DEVGUIDE.md

```diff
@@ -196,6 +196,17 @@ or
 > **Note**
 > Please note, that by default, **Release** version of IL baseline tests will be running in CI, so when updating baseline (.bsl) files, make sure to add `-c Release` flag to the build command.
 
+
+### Updating FCS surface area baselines
+
+```bash
+$env:TEST_UPDATE_BSL=1
```

Comment:
This is not needed for surface area, only test baselines (unless something changed lately, and if it does, it needs to be reverted to not require that flag).

Reply:
You sure?
https://github.com/dotnet/fsharp/blob/4c60c63c3a0fb3c7507ced9f2124966e67e7d71b/tests/FSharp.Compiler.Service.Tests/SurfaceArea.fs#L9-L18

Reply:
It used to not rely. That was the whole point of it, so when you run tests, you update it and it was part of PR and reflected there.

Reply:
It was always updating it, I am not sure why would we make it not to:

https://github.com/dotnet/fsharp/commit/4c60c63c3a0fb3c7507ced9f2124966e67e7d71b